migrations/migration-9/index.js

```diff
@@ -49,8 +49,8 @@ async function pinsToDatastore (blockstore, datastore, pinstore, onProgress) {
       pin.version = cid.version
     }
 
-    if (cid.codec !== 'dag-pb') {
-      pin.codec = multicodec.getNumber(cid.codec)
+    if (cid.code !== dagPb.code) {
+      pin.codec = cid.code
```

Comment:
is `pin.codec` always an integer? there's no risk of it being a string is there, because that's going to be unhappy in a few places below

Reply:
It's only ever been a number so we should be ok.